rotkehlchen/exchanges/ftx.py

```diff
@@ -0,0 +1,477 @@
+                )
+                log.error(
+                    'Error processing a FTX balance',
+                    error=msg,
+                )
+                continue
+
+        return dict(returned_balances), ''
+
+    def query_online_trade_history(
+            self,
+            start_ts: Timestamp,
+            end_ts: Timestamp,
+    ) -> List[Trade]:
+
+        raw_data = self._api_query('fills', start_time=start_ts, end_time=end_ts)
+        log.debug('FTX buys/sells history result', results_num=len(raw_data))
+
+        trades = []
+        for raw_trade in raw_data:
+            try:
+                trade = trade_from_ftx(raw_trade)
+            except UnknownAsset as e:
+                self.msg_aggregator.add_warning(
+                    f'Found FTX transaction with unknown asset '
+                    f'{e.asset_name}. Ignoring it.',
+                )
+                continue
+            except UnsupportedAsset as e:
+                self.msg_aggregator.add_warning(
+                    f'Found FTX trade with unsupported asset '
+                    f'{e.asset_name}. Ignoring it.',
+                )
+                continue
+            except (DeserializationError, KeyError) as e:
+                msg = str(e)
+                if isinstance(e, KeyError):
+                    msg = f'Missing key entry for {msg}.'
+                self.msg_aggregator.add_error(
+                    'Error processing a FTX trade. Check logs '
+                    'for details. Ignoring it.',
+                )
+                log.error(
+                    'Error processing a FTX trade',
+                    trade=raw_trade,
+                    error=msg,
+                )
+                continue
+
+            # limit coinbase trades in the requested time range here since there
+            # is no argument in the API call
+            if trade and trade.timestamp >= start_ts and trade.timestamp <= end_ts:
+                trades.append(trade)
+
+        return trades
+
+    def _deserialize_asset_movement(
+        self,
+        raw_data: Dict[str, Any],
+        movement_type: AssetMovementCategory,
+    ) -> Optional[AssetMovement]:
+        """Processes a single deposit/withdrawal from FTX and deserializes it
+
+        Can log error/warning and return None if something went wrong at deserialization
+        """
+        try:
+            if raw_data.get('status') not in ('complete', 'confirmed'):
+                return None
+
+            timestamp = deserialize_timestamp_from_date(raw_data['time'], 'iso8601', 'FTX')
+
+            amount = deserialize_asset_amount_force_positive(raw_data['size'])
+            asset = asset_from_ftx(raw_data.get('coin', ''))
+
+            # Only get address/transaction id for "send" type of transactions
+            address = None
+            transaction_id = None
+            fee = Fee(ZERO)
+
+            movement_category = movement_type
+            if raw_data.get('fee', None):
```

Comment:
please be explicit in checks.
```suggestion
            if raw_data.get('fee', None) is not None:
```